README.md

```diff
@@ -12,28 +12,28 @@ _Please reference [A Guide for the Samvera Community](https://samvera.github.io/
+| [questioning_authority](https://github.com/samvera/questioning_authority)             | [![Build Status](https://circleci.com/gh/samvera/questioning_authority.svg?style=svg)](https://circleci.com/gh/samvera/questioning_authority)             |                  |                  | 2.7.5            | 2.6.9            |                   | 6.1.1             | 6.0.2             | 5.2.4             |
+| [rubydora](https://github.com/samvera/rubydora)                                       | [![Build Status](https://circleci.com/gh/samvera/rubydora.svg?style=svg)](https://circleci.com/gh/samvera/rubydora)                                       |                  |                  | 2.7.5            | 2.6.9            |                   |                   | 6.0.2             | 5.2.4             |
+| [samvera.github.io](https://github.com/samvera/samvera.github.io)                     | [![Build Status](https://circleci.com/gh/samvera/samvera.github.io.svg?style=svg)](https://circleci.com/gh/samvera/samvera.github.io)                     | 3.1.1            | 3.0.3            | 2.7.5            | 2.6.9            | N/A               | N/A               | N/A               | N/A               |
+| [samvera-circleci-orb](https://github.com/samvera/samvera-circleci-orb)               | [![Build Status](https://circleci.com/gh/samvera/samvera-circleci-orb.svg?style=svg)](https://circleci.com/gh/samvera/samvera-circleci-orb)               | N/A              | N/A              | N/A              | N/A              | N/A               | N/A               | N/A               | N/A               |
+| [valkyrie](https://github.com/samvera/valkyrie)                                       | [![Build Status](https://circleci.com/gh/samvera/valkyrie.svg?style=svg)](https://circleci.com/gh/samvera/valkyrie)                                       |                  |                  | 2.7.5            | 2.6.5            |                   |                   | 6.0.5             | 5.2.8             |
+| [node-iiif](https://github.com/samvera/node-iiif)                                     | [![Build Status](https://circleci.com/gh/samvera/node-iiif.svg?style=svg)](https://circleci.com/gh/samvera/node-iiif)                                     | N/A              | N/A              | N/A              | N/A              | N/A               | N/A               | N/A               | N/A               |
+| [serverless-iiif](https://github.com/samvera/serverless-iiif)                         | [![Build Status](https://circleci.com/gh/samvera/serverless-iiif.svg?style=svg)](https://circleci.com/gh/samvera/serverless-iiif)                         | N/A              | N/A              | N/A              | N/A              | N/A               | N/A               | N/A               | N/A               |
```

Comment:
This table isn't rendering as expected, I don't think:

<img width="1176" alt="Screenshot 2022-12-07 at 2 41 10 PM" src="https://user-images.githubusercontent.com/65608/206280163-1c9d0fbb-d11e-49fe-9769-31af549294ba.png">